public/resources/stylesheets/page/_admin.scss

```diff
@@ -31,11 +31,18 @@
   margin: 0 10% 0 20%;
 }
 
+.body_wrapper {
+  margin: 0 10%;
+  width: 70vw;
+  height: auto;
+  overflow: hidden;
+}
+
 .content-table {
   border-collapse: collapse;
   margin: 25px 0;
   font-size: 0.9em;
-  min-width: 80rem;
+  min-width: 100%;
   border-radius: 5px 5px 0 0;
   overflow: hidden;
   box-shadow: rgba(50, 50, 93, 0.25) 0px 13px 27px -5px, rgba(0, 0, 0, 0.3) 0px 8px 16px -8px;
```

Comment:
save the colors in variables, and check if that color are not already in the variables file